### ISSUE TITLE
Линковка натягиваемого профиля обьема

### ISSUE THREAD

**Issue by h00lyshit** [SBProX/SBProX-Dev#1774]
Здравствуйте, уважаемые разработчики, очень хотелось бы чтобы натягиваемый профиль обьема линковался на графики других таймфреймов, в данный момент линкуются только линии, про линковку обьектов рисования тикет создавали ранее, но вот про профиль никто не писал, это дополнение очень бы упростило работу.

**Comment by ikki1561** [comment]
согласен, для более детального анализа многие смотрят меньший таймфрейм, и натягивают профиль для анализа.

**Comment by volfX6** [comment]
поддерживаю, доставляет неудобства, когда нанес профиль на старшем таймфрейме, а торгуешь на младшем, на младшем потом искать это место крутить график долго и мучительно, проще было бы если бы он наносился автоматически и на меньший фрейм, а взять например таких 3 профиля на 2 графиках это 10 минут дополнительного времени каждый раз, хочеться чтобы было как с линиями, нанес и забыл )

**Comment by moex4ange** [comment]
+1 надо

**Comment by bidask777** [comment]
удобно а то линковка линий есть а профиля нет

**Comment by Evgenii8** [comment]
**Добавьте в Линковку эти инструменты очень облегчит работу на разных тайм фреймах одного инструмента**   

 Линковка динамический профиль [ D ]
 Линковка линию [ L ]
 Линковка горизонтальную линию [ Left Ctrl + L ]
 Линковка трендовую линию [ Left Ctrl + T ]
 Линковка прямоугольник  [ R ]
 Линковка RANGE
 Линковка Эллипс [E]


![233](https://user-images.githubusercontent.com/73286473/103146506-dd147b80-478d-11eb-9b2a-ef68997e63a7.jpg)

**Давно просим вот этот комплект**
